django_bmemcached/memcached.py

```diff
@@ -30,21 +30,3 @@ def close(self, **kwargs):
         # Override base behavior of disconnecting from memcache on every HTTP request.
         # This method is, in practice, only called by Django on the request_finished signal
         pass
-
-    @property
-    def _cache(self):
-        client = getattr(self, '_client', None)
-        if client:
-            return client
-
-        if self._options:
-            client = self._lib.Client(
-                self._servers, self._options.get('username', None),
-                self._options.get('password', None)
-            )
-        else:
-            client = self._lib.Client(self._servers,)
-
-        self._client = client
-
-        return client
```

Comment:
The underlying property that this overrides is:
```python
@property
def _cache(self):
    if getattr(self, '_client', None) is None:
        self._client = self._lib.Client(self._servers, **self._options)
    return self._client
```

The main difference is that the superclass passes all the options through to the client initialisation, whereas this currently extracts username and password and discards the rest. By deleting the property it reverts to passing them all through, which is what we want (I think)?